packages/snaps-rpc-methods/src/endowments/cronjob.ts

```diff
@@ -116,13 +116,17 @@ export function validateCronjobCaveat(caveat: Caveat<string, any>) {
 
   const { value } = caveat;
 
-  if (!hasProperty(value, 'jobs') || !isPlainObject(value)) {
```

Comment:
Perhaps we should adjust `getCronjobCaveatMapper` to not add a caveat if the value is empty? Then we don't have to complicate the validation?